esearch -db nuccore -query "(txid130262[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid130262[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30700[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30700[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid908954[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid908954[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27723[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27723[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1545905[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1545905[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8162[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8162[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid43698[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid43698[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30876[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30876[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid29146[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid29146[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid195656[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid195656[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid490374[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid490374[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7900[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7900[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid107764[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid107764[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30972[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30972[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid54907[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid54907[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88651[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88651[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid134615[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid134615[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7850[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7850[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489899[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489899[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid205120[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid205120[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid84619[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid84619[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid143026[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid143026[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1072510[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1072510[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8202[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8202[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7934[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7934[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8279[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8279[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30988[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30988[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid241819[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid241819[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid2547680[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid2547680[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid990930[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid990930[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid178227[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid178227[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid82890[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid82890[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid83881[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid83881[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid303732[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid303732[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid170194[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid170194[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117867[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117867[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31017[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31017[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid316123[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid316123[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid163127[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid163127[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid36206[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid36206[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid143304[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid143304[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid69128[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid69128[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid461499[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid461499[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81361[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81361[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid149086[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid149086[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid150446[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid150446[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31024[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31024[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid490307[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid490307[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88658[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88658[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid463601[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid463601[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8214[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8214[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489799[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489799[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274806[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274806[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172132[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172132[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489833[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489833[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8065[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8065[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid94935[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid94935[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid178230[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid178230[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489942[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489942[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88661[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88661[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid56718[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid56718[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8253[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8253[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid36203[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid36203[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid582432[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid582432[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid412621[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid412621[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215350[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215350[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30757[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30757[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid181401[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid181401[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30850[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30850[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270592[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270592[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30908[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30908[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7865[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7865[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31098[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31098[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8157[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8157[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid181390[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid181390[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7805[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7805[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7802[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7802[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215353[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215353[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid181420[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid181420[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid206122[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid206122[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489924[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489924[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid163122[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid163122[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117924[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117924[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid412645[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid412645[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8184[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8184[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270595[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270595[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid206106[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid206106[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88666[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88666[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57770[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57770[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30828[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30828[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270610[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270610[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid29142[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid29142[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30806[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30806[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30910[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30910[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid181415[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid181415[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid76917[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid76917[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489927[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489927[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid206140[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid206140[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7869[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7869[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid94304[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid94304[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid82793[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid82793[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27583[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27583[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid118146[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid118146[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid68511[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid68511[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8113[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8113[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid76914[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid76914[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30943[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30943[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57768[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57768[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid55118[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid55118[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid391217[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid391217[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid178225[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid178225[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7941[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7941[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27766[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27766[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8092[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8092[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270598[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270598[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30888[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30888[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid2823367[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid2823367[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8101[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8101[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid556237[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid556237[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30947[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30947[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8076[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8076[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1545868[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1545868[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid94922[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid94922[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57772[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57772[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117914[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117914[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30469[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30469[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid118158[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid118158[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid412631[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid412631[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid390322[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid390322[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid390325[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid390325[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid428439[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid428439[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31034[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31034[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88680[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88680[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215389[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215389[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid75380[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid75380[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid365061[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid365061[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid173245[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid173245[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7832[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7832[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid166260[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid166260[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid86197[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid86197[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7926[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7926[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid50791[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid50791[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid181462[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid181462[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid43062[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid43062[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215365[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215365[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid75381[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid75381[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid223796[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid223796[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30974[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30974[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27718[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27718[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117921[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117921[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid630724[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid630724[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid55115[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid55115[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172143[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172143[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid76072[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid76072[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid206152[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid206152[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid28756[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid28756[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8045[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8045[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1545873[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1545873[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid130264[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid130264[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid69291[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid69291[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27767[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27767[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid2748623[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid2748623[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274463[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274463[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid565091[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid565091[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid242956[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid242956[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172114[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172114[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30507[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30507[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid603452[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid603452[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid443779[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid443779[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid2800422[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid2800422[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid178224[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid178224[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid63826[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid63826[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8220[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8220[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30764[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30764[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid48439[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid48439[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid2025516[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid2025516[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274798[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274798[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31099[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31099[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid3349725[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid3349725[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid86359[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid86359[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30840[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30840[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid143897[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid143897[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid43254[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid43254[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid376643[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid376643[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88694[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88694[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid40580[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid40580[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid337659[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid337659[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid556143[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid556143[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7790[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7790[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30983[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30983[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7855[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7855[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117842[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117842[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid242959[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid242959[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid47697[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid47697[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30986[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30986[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid390339[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid390339[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117889[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117889[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid760215[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid760215[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid166254[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid166254[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215384[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215384[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172118[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172118[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489917[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489917[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27768[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27768[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30842[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30842[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215393[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215393[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30843[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30843[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8247[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8247[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57769[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57769[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30845[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30845[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81368[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81368[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1204718[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1204718[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8184[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8184[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7895[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7895[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid97158[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid97158[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30846[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30846[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30847[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30847[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid376640[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid376640[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid390344[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid390344[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30848[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30848[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid2743726[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid2743726[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid242963[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid242963[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid183715[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid183715[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30849[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30849[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8071[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8071[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81371[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81371[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31002[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31002[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81641[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81641[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30850[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30850[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30887[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30887[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30761[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30761[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8061[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8061[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30851[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30851[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid3051911[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid3051911[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57976[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57976[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7930[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7930[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88706[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88706[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid181423[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid181423[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30758[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30758[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30852[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30852[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8061[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8061[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid170197[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid170197[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57977[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57977[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31029[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31029[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid143346[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid143346[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88673[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88673[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30853[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30853[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid556238[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid556238[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30759[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30759[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid118143[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid118143[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid42148[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid42148[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8189[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8189[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30854[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30854[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7944[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7944[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid46660[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid46660[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30760[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30760[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid68515[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid68515[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1158979[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1158979[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid556236[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid556236[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7762[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7762[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid40665[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid40665[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117892[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117892[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30856[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30856[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid118167[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid118167[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30857[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30857[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid412628[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid412628[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88699[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88699[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274689[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274689[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid118175[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid118175[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489944[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489944[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215399[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215399[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30975[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30975[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid55111[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid55111[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172126[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172126[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8206[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8206[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid241355[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid241355[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7802[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7802[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215327[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215327[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid242930[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid242930[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid54912[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid54912[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid94930[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid94930[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid170204[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid170204[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270589[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270589[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30858[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30858[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid168096[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid168096[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31101[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31101[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8012[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8012[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31025[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31025[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid428447[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid428447[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117915[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117915[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7999[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7999[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172138[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172138[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1003977[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1003977[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid171414[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid171414[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1003985[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1003985[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1213740[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1213740[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid509847[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid509847[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31102[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31102[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid178226[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid178226[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid215337[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid215337[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30859[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30859[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30860[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30860[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270605[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270605[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid178231[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid178231[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7746[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7746[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270656[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270656[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid56724[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid56724[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid390366[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid390366[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid210573[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid210573[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid163113[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid163113[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid206137[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid206137[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30987[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30987[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117848[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117848[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid170200[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid170200[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30955[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30955[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid178228[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid178228[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117840[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117840[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30861[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30861[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8256[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8256[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30997[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30997[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8079[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8079[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid490375[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid490375[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81383[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81383[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30917[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30917[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7911[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7911[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid55700[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid55700[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid112726[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid112726[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30862[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30862[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30863[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30863[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30864[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30864[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid86367[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid86367[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30865[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30865[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid55129[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid55129[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid260504[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid260504[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid55133[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid55133[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid263714[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid263714[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1077316[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1077316[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30949[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30949[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1490017[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1490017[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57978[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57978[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30866[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30866[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid160493[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid160493[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172661[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172661[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172135[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172135[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30976[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30976[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30890[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30890[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30867[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30867[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1545867[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1545867[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30475[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30475[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8045[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8045[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81387[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81387[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid130260[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid130260[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274754[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274754[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid259919[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid259919[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117884[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117884[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7860[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7860[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid134992[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid134992[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid490374[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid490374[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid86200[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid86200[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid28771[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid28771[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid88709[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid88709[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid136488[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid136488[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid182234[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid182234[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8015[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8015[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid195629[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid195629[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8247[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8247[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30869[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30869[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30918[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30918[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30870[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30870[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid94935[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid94935[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8224[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8224[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30885[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30885[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid433665[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid433665[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid172121[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid172121[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid52902[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid52902[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8108[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8108[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7826[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7826[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30891[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30891[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274692[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274692[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30871[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30871[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid87098[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid87098[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274712[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274712[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid75382[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid75382[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30874[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30874[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30948[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30948[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270524[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270524[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117916[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117916[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8169[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8169[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27769[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27769[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid376648[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid376648[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30511[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30511[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid55140[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid55140[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid378069[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid378069[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid500384[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid500384[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid473342[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid473342[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid68500[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid68500[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30892[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30892[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid123351[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid123351[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid94243[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid94243[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid409994[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid409994[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid722446[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid722446[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid274706[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid274706[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid118152[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid118152[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid72045[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid72045[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81390[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81390[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid238703[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid238703[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30875[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30875[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid242970[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid242970[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid316148[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid316148[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31031[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31031[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid210577[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid210577[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489881[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489881[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid412640[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid412640[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30480[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30480[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid96776[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid96776[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid56735[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid56735[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81393[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81393[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31027[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31027[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31026[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31026[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid7839[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid7839[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27770[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27770[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid148605[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid148605[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid270659[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid270659[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27774[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27774[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid300225[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid300225[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid57771[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid57771[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid70846[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid70846[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid40662[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid40662[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid86673[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid86673[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid81374[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid81374[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8243[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8243[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid75383[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid75383[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30984[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30984[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid117845[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid117845[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30893[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30893[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid31103[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid31103[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid1489918[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid1489918[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8193[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8193[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid56259[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid56259[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid42164[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid42164[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid28683[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid28683[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8830[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8830[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8899[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8899[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30558[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30558[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9765[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9765[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid85103[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid85103[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8907[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8907[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8465[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8465[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8493[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8493[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9726[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9726[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid27792[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid27792[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid37067[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid37067[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid29135[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid29135[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8602[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8602[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30443[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30443[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid43314[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid43314[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid37576[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid37576[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid184169[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid184169[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid85545[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid85545[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8910[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8910[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid226940[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid226940[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9702[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9702[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid56260[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid56260[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30444[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30444[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30451[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30451[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30445[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30445[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9206[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9206[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9709[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9709[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9740[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9740[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9750[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9750[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30448[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30448[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30451[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30451[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid425629[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid425629[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid66056[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid66056[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid8917[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid8917[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9231[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9231[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid54049[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid54049[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid30446[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid30446[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid33574[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid33574[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
esearch -db nuccore -query "(txid9756[ORGN]) AND (Small subunit [Title] OR 12S[Title] OR 12S ribosomal RNA[Title] OR 12S rRNA[Title]) NOT environmental sample[Title] NOT environmental samples[Title] NOT environmental[Title] NOT uncultured[Title] NOT unclassified[Title] NOT unidentified[Title] NOT unverified[Title] " | efetch -format fasta >> 12s_fish_nuccore.fasta
esearch -db gene -query '(txid9756[ORGN]) AND (Small subunit[Title] OR 12S[Title]) AND ("source mitochondrion"[property] AND alive[prop])' | efetch -format docsum |   xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop | while IFS=$'\t' read acn str stp;   do   efetch -db nuccore -format fasta  -id "$acn" -chr_start "$str" -chr_stop "$stp"; done | sed "s/>/>12S_/" >> 12s_fish_gene.fasta
